--- /usr/local/miniconda3/envs/agentgym-rl/lib/python3.10/site-packages/vllm/model_executor/models/registry.py.pre_qwen3	2026-09-19 08:00:57.079129708 +0800
+++ /usr/local/miniconda3/envs/agentgym-rl/lib/python3.10/site-packages/vllm/model_executor/models/registry.py	2026-09-19 08:00:57.099129709 +0800
@@ -73,6 +73,7 @@
     "PhiMoEForCausalLM": ("phimoe", "PhiMoEForCausalLM"),
     # QWenLMHeadModel supports multimodal
     "Qwen2ForCausalLM": ("qwen2", "Qwen2ForCausalLM"),
+    "Qwen3ForCausalLM": ("qwen3", "Qwen3ForCausalLM"),  # 本地移植 2026-09-19
     "Qwen2MoeForCausalLM": ("qwen2_moe", "Qwen2MoeForCausalLM"),
     "RWForCausalLM": ("falcon", "FalconForCausalLM"),
     "StableLMEpochForCausalLM": ("stablelm", "StablelmForCausalLM"),
